server/chat/backend/agent/tools/notion/postmortem.py

```
@@ -0,0 +1,606 @@
+
+    page = client.create_page(
+        parent={"database_id": database_id}, properties=properties
+    )
+    page_id = page.get("id")
+    page_url = page.get("url")
+    if not page_id:
+        raise ValueError("Notion create_page returned no page id")
+
+    try:
+        client.update_page_markdown(page_id, content, mode="replace")
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)